dstep/translator/Output.d

```diff
@@ -1062,14 +1060,13 @@ class ClassData : StructData
     string superclass;
     string[] interfaces;
 
-    HashSet!(string) propertyList;
+    bool[string] propertyList;
```

Comment:
In `clang/Util.d:82` there is defined a type `Set` which is already used in some other places.